packages/core/src/components/Table/stories/TableRenderers/TableRenderers.stories.tsx

```diff
@@ -30,6 +30,12 @@ import {
 
 import { makeRenderersData, NewRendererEntry } from "../storiesUtils";
 
+export default {
```

Comment:
Do we need this? We already have `TableRenderers.stories.mdx`. 
This is creating a new "Samples" page inside the "Table Renderers" folder repeating the samples we already have inside the "Renderers" folder.

Reply:
`.stories.tsx` must have a meta default export with meta 😕
removing `.stories` from name